RecoLocalTracker/SiPixelRecHits/plugins/PixelCPETemplateRecoESProducer.cc

```diff
@@ -66,4 +67,18 @@ std::unique_ptr<PixelClusterParameterEstimator> PixelCPETemplateRecoESProducer::
                                                 &iRecord.get(templateDBobjectToken_));
 }
 
+void PixelCPETemplateRecoESProducer::fillDescriptions(edm::ConfigurationDescriptions& descriptions) {
+  edm::ParameterSetDescription desc;
+
+  // from PixelCPEBase
+  PixelCPEBase::fillPSetDescription(desc);
+
+  // from PixelCPETemplateReco
+  PixelCPETemplateReco::fillPSetDescription(desc);
+
+  // specific to PixelCPETemplateRecoESProducer
+  desc.add<std::string>("ComponentName", "PixelCPETemplateReco");
+  descriptions.add("templates", desc);
```

Comment:
due to a name conflict with PixelCPETemplateReco_cfi.py, this should probably be named ```_templates_default``` so that it can be loaded in PixelCPETemplateReco_cfi and era-modified there as well.

Reply:
done.